### PR TITLE
Atos connector updated. New connectors added.

### PR DESCRIPTION
**Pull request by Netmisa**
# Introduction

New operators added:
- Keolis
- Effia
- Effia transport
- Kéolis
# How to test
## 1. Run unit tests

_Execute command:_

``` shell
JORMUNGANDR_CONFIG_FILE="<navitia_path>/run/jormungandr_settings.py" PYTHONPATH="<navitia_path>/source/navitiacommon:<navitia_path>/source/jormungandr" KRAKEN_BUILD_DIR=<navitia_path>/bin/release  py.test parking_space_availability/bss/tests/atos_test.py 
```

### CODE REVIEW

File: source/jormungandr/jormungandr/parking_space_availability/bss/tests/atos_test.py [hunk 1]
```diff
@@ -53,9 +53,17 @@ def parking_space_availability_atos_support_poi_test():
     """
     provider = AtosProvider(u'10', u'Vélitul', u'https://webservice.atos.com?wsdl')
     assert provider.support_poi(poi)
+    poi['properties']['operator'] = 'EFFIA'
+    assert provider.support_poi(poi)
+    poi['properties']['operator'] = 'Keolis'
+    assert provider.support_poi(poi)
+    poi['properties']['operator'] = 'EFFIA Transport'
+    assert provider.support_poi(poi)
+    poi['properties']['operator'] = u'Kéolis'
```
Comment (xlqian) [inline source/jormungandr/jormungandr/parking_space_availability/bss/tests/atos_test.py:62]: can you add test for 'KÉOLIS' plz?

**Comment by xlqian** [comment]
Ok for me

File: source/jormungandr/jormungandr/parking_space_availability/bss/atos.py [hunk 2]
```diff
@@ -37,7 +38,7 @@
 
 class AtosProvider(BssProvider):
 
-    OPERATOR = 'Keolis'
+    OPERATORS = ('keolis', 'effia', 'effia transport', u'kéolis')
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/parking_space_availability/bss/atos.py:41]: it's really minor, but if this list is meant to grow, I think it's better to use a set
=> `OPERATORS = {'keolis', 'effia', 'effia transport', u'kéolis'}`

Comment (kinnou02) [inline source/jormungandr/jormungandr/parking_space_availability/bss/atos.py:41]: This should be in configuration, and don't forget the jcedaux connector too.

File: source/jormungandr/jormungandr/parking_space_availability/bss/atos.py [hunk 3]
```diff
@@ -51,7 +52,7 @@ def __repr__(self):
 
     def support_poi(self, poi):
         properties = poi.get('properties', {})
-        return properties.get('operator') == self.OPERATOR and \
+        return properties.get('operator').lower() in self.OPERATORS and \
                properties.get('network') == self.network
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/parking_space_availability/bss/atos.py:56]: By the way, a `lower()` on the network would help too.

**Pull request closed without merging**